autoload/lsp.vim

```diff
@@ -18,6 +18,18 @@ let s:notification_callbacks = [] " { name, callback }
 "    }
 let s:file_content = {}
 
+" incr if s:servers update
+let s:update_cnt = 0
```

Comment:
would be good to have a proper name here.

Reply:
renames to s:servers_update_cnt